### ISSUE TITLE
Integrate domain check feature [Hacktoberfest]

### ISSUE THREAD

**Issue by MadhavBahlMD** [MadhavBahlMD/startup-name-generator#3]
Clicking on a proposed name takes the user to another screen where the user can see availability of that domain

To keep things simple, as of now we can probably open some website like https://namechk.com/ and pre-fill the domain name

**Comment by prathameshmhatre** [comment]
@MadhavBahlMD  can I work on the Issue by using  [www.namecheap.com](www.namecheap.com)? If you allow I will start working on this Issue.

**Comment by MadhavBahlMD** [comment]
Sure @prathameshmhatre , go ahead 🥳 

Yes, namecheap looks good, probably you will open this URL - `https://www.namecheap.com/domains/registration/results/?domain=madhavbahl` ??

Looking forward to your PR

**Comment by MadhavBahlMD** [comment]
Also, @prathameshmhatre , if you want you can open the PR on 1st October to make it count as a hacktoberfest contribution

**Comment by prathameshmhatre** [comment]
Cool thanks,  Will work on this and send you the PR on 1st October. Can you please assign the task to me :)

**Comment by MadhavBahl** [comment]
@prathameshmhatre any updates?

**Comment by prathameshmhatre** [comment]
Yes, Give me a day or two will create the PR sorry for the delay :)

**Comment by AishwaryaChat** [comment]
@MadhavBahl  I see this issue is still open, do you think I can work on it? it will be my first ever contribution for open source

**Comment by MadhavBahl** [comment]
Sure @AishwaryaChat , I'll assign this issue to you
All the bet for your first open source contribution, please let me know if I can help in any way.
Thanks!
